lib/assets/javascripts/_modules/search.js

```diff
@@ -49,34 +56,25 @@
     }
 
     function attach () {
-      // Search functionality on search text input
-      $searchInput.on('input', function (e) {
+      // Set focus on the results close link instead of submiting the search
+      // form to Google
+      $searchForm.on('submit', function (e) {
         e.preventDefault()
-        query = $(this).val()
+
+        query = $searchInput.val()
         s.search(query, function (r) {
           results = r
           renderResults(query)
           updateTitle()
         })
         if (window.ga) {
-          window.clearTimeout(queryTimer)
-          queryTimer = window.setTimeout(sendQueryToAnalytics, 1000)
+          sendQueryToAnalytics()
         }
-      })
 
-      // Set focus on the first search result instead of submiting the search
-      // form to Google
-      $searchForm.on('submit', function (e) {
-        e.preventDefault()
         showResults()
-        var $firstResult = $searchResults.find('.search-result__title a').first()
-        if ($firstResult.length) {
-          $firstResult.focus()
-        } else {
-          // if there are no results, show the "0 results" state
-          results = []
-          updateTitle()
-        }
+        var $closeButton = $searchResultsWrapper.find('.search-results__close')
+        $closeButton.focus()
```

Comment:
I think we've already got a variable that references this element we can use here?

```suggestion
        $searchResultsClose.focus()
```